src/argilla/server/apis/v1/handlers/datasets.py

```diff
@@ -527,24 +547,26 @@ async def create_dataset_metadata_property(
     db: AsyncSession = Depends(get_async_db),
     search_engine: SearchEngine = Depends(get_search_engine),
     dataset_id: UUID,
-    metadata_prop_create: MetadataPropertyCreate,
+    metadata_property_create: MetadataPropertyCreate,
```

Comment:
We should have an update endpoint to allow users change the property visibility (`visible_for_annotators` attribute)  Not sure if should tackled here or in a separate PR @jfcalvo

Reply:
I will take care of that new update endpoint.